nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -36,6 +36,8 @@
 
 - `fileSystems.<name>.autoResize` now uses `systemd-growfs` to resize the file system online in stage 2. This means that `f2fs` and `ext2` can no longer be auto resized, while `xfs` and `btrfs` now can be.
 
+- The executable file names for `firefox-devedition`, `firefox-beta`, `firefox-esr` now matches their package names, which is consistent with the `firefox-*-bin` packages. The desktop entries are also updated so that you can have mulitple editions of firefox in your app launcher.
+
```

Comment:
This would need to be moved to a different release now that 23.11 is out. Might be best to just leave release notes for a separate PR?

Reply:
Done